### PR TITLE
Use `anon_chi` where possible

### PR DESCRIPTION
**Pull request by Jennit07**
Opening as a draft for now so we can see the commit history

### CODE REVIEW

File: R/process_extract_prescribing.R [hunk 1]
```diff
@@ -38,7 +38,8 @@ process_extract_prescribing <- function(data, year, write_to_disk = TRUE) {
       record_keydate2 = .data$record_keydate1,
       # Add SMR type variable
       smrtype = add_smrtype(.data$recid)
-    )
+    ) %>%
+    slfhelper::get_anon_chi()
```
Comment (Jennit07) [inline R/process_extract_prescribing.R:42]:
```suggestion
    ) 
```
Keep IT extracts as chi for now

File: R/read_extract_prescribing.R [hunk 2]
```diff
@@ -6,7 +6,7 @@
 read_extract_prescribing <- function(year, file_path = get_it_prescribing_path(year)) {
   pis_file <- read_file(file_path,
     col_type = readr::cols_only(
-      "Pat UPI [C]" = readr::col_character(),
+      "anon_chi" = readr::col_character(),
```
Comment (Jennit07) [inline R/read_extract_prescribing.R:9]:
```suggestion
      "Pat UPI [C]" = readr::col_character(),
```
Keep IT extracts read in as chi for now

File: R/read_extract_prescribing.R [hunk 3]
```diff
@@ -17,14 +17,15 @@ read_extract_prescribing <- function(year, file_path = get_it_prescribing_path(y
   ) %>%
     # Rename variables
     dplyr::rename(
-      chi = "Pat UPI [C]",
+      anon_chi = "anon_chi",
```
Comment (Jennit07) [inline R/read_extract_prescribing.R:20]:
```suggestion
      chi = "Pat UPI [C]",
```

File: R/read_extract_prescribing.R [hunk 4]
```diff
@@ -17,14 +17,15 @@ read_extract_prescribing <- function(year, file_path = get_it_prescribing_path(y
...
       dob = "Pat DoB [C]",
       gender = "Pat Gender",
       postcode = "Pat Postcode [C]",
       gpprac = "Practice Code",
       no_paid_items = "Number of Paid Items",
       cost_total_net = "PD Paid GIC excl. BB"
-    )
+    ) %>%
+    slfhelper::get_chi()
```
Comment (Jennit07) [inline R/read_extract_prescribing.R:28]:
```suggestion
    ) 
```

File: R/read_it_chi_deaths.R [hunk 5]
```diff
@@ -9,16 +9,17 @@
 read_it_chi_deaths <- function(file_path = get_it_deaths_path()) {
   it_chi_deaths <- read_file(file_path,
     col_type = readr::cols(
-      "PATIENT_UPI [C]" = readr::col_character(),
+      "anon_chi" = readr::col_character(),
```
Comment (Jennit07) [inline R/read_it_chi_deaths.R:12]:
```suggestion
      "PATIENT_UPI [C]" = readr::col_character(),
```

File: R/read_it_chi_deaths.R [hunk 6]
```diff
@@ -9,16 +9,17 @@
...
       "PATIENT DoD DATE (NRS)" = readr::col_date(format = "%d-%m-%Y"),
       "PATIENT DoD DATE (CHI)" = readr::col_date(format = "%d-%m-%Y")
     )
   ) %>%
     dplyr::rename(
-      chi = "PATIENT_UPI [C]",
+      anon_chi = "anon_chi",
```
Comment (Jennit07) [inline R/read_it_chi_deaths.R:18]:
```suggestion
chi = "PATIENT_UPI [C]",
```

File: R/read_it_chi_deaths.R [hunk 7]
```diff
@@ -9,16 +9,17 @@
...
       death_date_nrs = "PATIENT DoD DATE (NRS)",
       death_date_chi = "PATIENT DoD DATE (CHI)"
-    )
+    ) %>%
+    slfhelper::get_chi()
```
Comment (Jennit07) [inline R/read_it_chi_deaths.R:22]:
```suggestion
    ) 
```

File: R/read_lookup_sc_demographics.R [hunk 8]
```diff
@@ -27,6 +27,7 @@ read_lookup_sc_demographics <- function(sc_dvprod_connection = phs_db_connection
 
   if (!fs::file_exists(get_sandpit_extract_path(type = "demographics"))) {
     sc_demog %>%
+      slfhelper::get_anon_chi(chi = "chi_upi") %>%
```
Comment (Jennit07) [inline R/read_lookup_sc_demographics.R:30]:
```suggestion
      slfhelper::get_anon_chi(chi_var = "chi_upi") %>%
```

File: R/create_episode_file.R [hunk 9]
```diff
@@ -438,7 +441,7 @@ join_cohort_lookups <- function(
 #' @param file_type episode or individual file
 join_sc_client <- function(data,
                            year,
-                           sc_client = read_file(get_sc_client_lookup_path(year)),
+                           sc_client = read_file(get_sc_client_lookup_path(year) %>% slfhelper::get_chi()),
```
Comment (Jennit07) [inline R/create_episode_file.R:444]:
```suggestion
                           sc_client = read_file(get_sc_client_lookup_path(year)) %>% slfhelper::get_chi()),
```

File: R/create_episode_file.R [hunk 10]
```diff
@@ -36,6 +36,7 @@ create_episode_file <- function(
   processed_data_list <- purrr::discard(processed_data_list, ~ is.null(.x) | identical(.x, tibble::tibble()))
```
Comment (Jennit07) [inline R/create_episode_file.R:36]: slf_deaths_lookup = read_file(get_slf_deaths_lookup_path(year)) %>%
slfhelper::get_chi()

**Review by lizihao-anu — APPROVED** [review]

**Pull request merged**